code/modules/admin/verbs/adminweather.dm

```diff
@@ -0,0 +1,78 @@
+ADMIN_VERB(run_weather, R_ADMIN|R_FUN, "Run Weather", "Triggers specific weather on the z-level you choose.", ADMIN_CATEGORY_WEATHER)
+
+	var/static/list/weather_choices = list()
+	if(!length(weather_choices))
+		for(var/datum/weather/weather_type as anything in subtypesof(/datum/weather))
+			weather_choices[initial(weather_type.type)] = weather_type
```

Comment:
s no reason for this to be static really the list is gonna be real short

Reply:
Agree

Reply:
Applied via bb83bf3c67b0f4da41885f8e3fa59ba7056fa051